### PR TITLE
Add in new tutorials

### LINKED ISSUE #125: Check and update tutorials
Closes #112

Some Web examples would be better suited to be in tutorials. (see 'temp' folder). In addition to adding them, make sure the documentation is simple and complete. 

- [x] configuration.js
- [x] simplestexample.html
- [x] standalone example
- [x] publishing your own library

### PR DESCRIPTION
**Pull request by eirizarry**
Closes #125 

This PR adds in some new tutorials that had previously been on the Web examples page. 

They include: 
- Configuration
- Simplest Example
- Hosting Locally (Renamed from Standalone Example)

### CODE REVIEW

File: content/web/tutorials/standalone-example.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    var blueMarble = new WorldWind.BMNGRestLayer(null, "../standalonedata/Earth/BlueMarble256/");
+    wwd.addLayer(blueMarble);
+
+    // Add the REST Landsat layer (this offline data covers only a sector of the globe in North America)
+    var landSat = new WorldWind.LandsatRestLayer(null, "../standalonedata/Earth/Landsat");
+    wwd.addLayer(landSat);
+
+---
+
+You can See this tutorial in action through the [Standalone Example](https://files.worldwind.arc.nasa.gov/artifactory/apps/web/examples/Standalone.html).
```
Comment (pdavidc) [inline content/web/tutorials/standalone-example.md]: Minor typo: need a lowercase S

File: content/web/tutorials/configuration.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+before you create the WorldWindow or any other WorldWind object.
+
+---
+
+## Setting the Logging Level
+
+    // Get detailed logging
+    WorldWind.Logger.setLoggingLevel(WorldWind.Logger.LEVEL_WARNING);
+
+    //Turn off logging
```
Comment (pdavidc) [inline content/web/tutorials/configuration.md]: Missing space after //

File: content/web/tutorials/standalone-example.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+You can also use this [example data](https://worldwind32.arc.nasa.gov/WebWorldWind-StandaloneData.tar.gz) from NASA.
+
+---
+
+### The Script
+
+Add the WorldWind library:
+
+    <script src="https://YOUR_SERVER/worldwind.min.js" type="text/javascript"></script>
```
Comment (pdavidc) [inline content/web/tutorials/standalone-example.md]: Best to make this this element self-closing: <script src="..."/>

File: content/web/tutorials/simplest-example.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+This step-by-step tutorial illustrates how to use Web WorldWind in an HTML file.
+
+---
+
+### Breakdown
+
+First, in the head tag of your HTML, add a script element to include the Web WorldWind library.
+
+    <script src="https://files.worldwind.arc.nasa.gov/artifactory/apps/web/worldwind.min.js" type="text/javascript"></script>
```
Comment (pdavidc) [inline content/web/tutorials/simplest-example.md]: Best to make this this element self-closing: <script src="..."/>

File: content/web/tutorials/simplest-example.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+
+Next, create an HTML5 canvas, defining its width and heighth.
+
+    <canvas id="canvasOne" width="1024" height="768">
```
Comment (pdavidc) [inline content/web/tutorials/simplest-example.md]: Best to make this this element self-closing: <canvas id="..."/>

File: content/web/tutorials/standalone-example.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+Create an HTML5 canvas:
+
+    <canvas id="canvasOne" width="1024" height="768">
```
Comment (pdavidc) [inline content/web/tutorials/standalone-example.md]: Best to make this this element self-closing: <canvas id="..."/>

File: content/web/tutorials/simplest-example.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+
+            // Add a compass, a coordinates display and some view controls to the WorldWindow.wwd.addLayer(new WorldWind.CompassLayer());
+            wwd.addLayer(new WorldWind.CoordinatesDisplayLayer(wwd));
+            wwd.addLayer(new WorldWind.ViewControlsLayer(wwd));
+        }
+    </script>
+    </body>
+    </html>
+
+If you run this example, the result is a simple, interactive [globe](https://files.worldwind.arc.nasa.gov/artifactory/apps/web/examples/SimplestExample.html).
```
Comment (pdavidc) [inline content/web/tutorials/simplest-example.md]: The Gestures section is easy to overlook. What if we renamed Gestures to something like Runtime Result, and moved this sentence and its link into the renamed section?

File: content/web/tutorials/standalone-example.md [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+### The Data
+
+Get the [latest release](https://github.com/NASAWorldWind/WebWorldWind/releases/tag/v0.9.0-RC2) on GitHub.
+
+Host the WorldWind library and the images folder on your HTTP server:
+
+    https://YOUR_SERVER/worldwind.min.js
+    https://YOUR_SERVER/image/
+
+Get some data and unpack to:
```
Comment (pdavidc) [inline content/web/tutorials/standalone-example.md:22]: Clarification: the standalone folder is specific to the example data from NASA. With custom data, the folder could be named anything.

File: content/web/tutorials/standalone-example.md [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+listdescription: Describes the how to host Web WorldWind on your own HTTP server.
+---
+
+## Hosting Locally
+
+---
+
+### The Data
+
+Get the [latest release](https://github.com/NASAWorldWind/WebWorldWind/releases/tag/v0.9.0-RC2) on GitHub.
```
Comment (pdavidc) [inline content/web/tutorials/standalone-example.md]: We'll want links going to https://github.com/NASAWorldWind/WebWorldWind/releases/latest

File: content/web/tutorials/simplest-example.md [hunk 10]
```diff
@@ -0,0 +1,96 @@
...
+with various image and control layers.
+
+---
+
+### Putting it All Together
+
+Here is what it looks like in a working example:
+
+    <!DOCTYPE html>
+        <!-- This is a very simple example of using Web WorldWind. -->
```
Comment (pdavidc) [inline content/web/tutorials/simplest-example.md]: Minor typo: too many indentations

**Review by pdavidc — CHANGES_REQUESTED** [review]

File: content/web/tutorials/simplest-example.md (hunk 5 above)
Comment (eirizarry) [reply content/web/tutorials/simplest-example.md]: The canvas element also contains "Your browser does not support HTML5 Canvas" between the opening and closing tags.

**Review by pdavidc — APPROVED** [review]

**Pull request merged**